masonry/src/widget/align.rs

```diff
@@ -85,17 +85,11 @@ impl Align {
 
 // --- MARK: IMPL WIDGET ---
 impl Widget for Align {
-    fn on_pointer_event(&mut self, ctx: &mut EventCtx, event: &PointerEvent) {
-        self.child.on_pointer_event(ctx, event);
-    }
+    fn on_pointer_event(&mut self, _ctx: &mut EventCtx, _event: &PointerEvent) {}
```

Comment:
I'm split. On the one hand, some widgets won't need to implement them. On the other hand, filling in default implementations take like two minutes, and it's good to be mindful about what methods you implement. I lean towards "no unless >95% of widgets are expected not to implement the method".

Reply:
On the other hand, it is a bunch of boilerplate and it means you can't search for meaningful implementations of the method rather than "all implementations, even the empty ones".

Reply:
Hmm I also think default implementations may reduce unnecessary clutter and I think are one of the main advantages of avoiding the hierarchical child-calling we had previously.
Instead I think good documentation should help the implementer of that trait what they actually need to implement for their widget.

Reasons for default impls get even more vaild, when we add more of these methods to the trait (which I'm pretty sure will happen). So we could extend the trait more deliberately without having to think about all the boilerplate or breaking changes that it causes.

Reply:
I'm not so sure, but I guess it doesn't hurt to add a default impl.

Empty implementations can be removed in a follow-up PR.